flow/project.py

```diff
@@ -405,11 +414,13 @@ def __init__(self, cmd, pre=None, post=None, directives=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def eligible(self, job):
+    def eligible(self, job, ignore_condition=Ignore_condition.NONE):
```

Comment:
```suggestion
    def eligible(self, job, ignore_conditions=Ignore_condition.NONE):
```
Since this is applied to *all* conditions, right?

Reply:
I agree, thanks for pointing this out.